packages/build-deb-packages.sh

```diff
@@ -56,26 +61,26 @@ ex cp README.md $tmpdir/gdrcopy/debian/README.source
 ex rm -f $tmpdir/gdrcopy_${VERSION}.orig.tar.gz
 
 ex cd $tmpdir/gdrcopy
-ex find . -type f -exec sed -i "s/@VERSION@/${VERSION}/g" {} +
+ex find . -type f -exec sed -i "s/@VERSION@/${FULL_VERSION}/g" {} +
 
 ex cd $tmpdir
 ex mv gdrcopy gdrcopy-${VERSION}
 ex tar czvf gdrcopy_${VERSION}.orig.tar.gz gdrcopy-${VERSION}
 
 ex cd $tmpdir/gdrcopy-${VERSION}
-ex debuild --set-envvar=CUDA=$CUDA -us -uc
+ex debuild --set-envvar=CUDA=${CUDA} --set-envvar=PKG_CONFIG_PATH=${PKG_CONFIG_PATH} -us -uc
```

Comment:
does this works even when PKG_CONFG_PATH is not set by the user?

Reply:
Yes, it works fine. To be sure, can you test it on gc11 as well? The script should run successfully without setting PKG_CONFIG_PATH.

Reply:
it builds ok there